utils/ml.py

```diff
@@ -13,7 +13,7 @@ def extract_features(df):
     # Extract show name and season from title
     df['show'] = df['Title'].apply(lambda x: x.split(':')[0] if ':' in x else x)
     df['season'] = df['Title'].apply(lambda x: 
-        int(re.search(r'Season (\d+)', x).group(1)) if 'Season' in x else 0)
+        int(re.search(r'Season (\d+)', x).group(1)) if re.search(r'Season (\d+)', x) else 0)
```

Comment:
Note, this change is due to issues with a show in my list that was "XX Seasonal YY" -> This was causing errors.